app/build.gradle

```diff
@@ -3,24 +3,50 @@ apply plugin: 'com.google.gms.google-services'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin: 'com.google.firebase.appdistribution'
+apply plugin: "com.github.triplet.play"
+apply from: "../scripts/versions.gradle"
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
     defaultConfig {
         applicationId "jp.co.soramitsu.fearless"
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
-        versionCode rootProject.versionCode
-        versionName rootProject.versionName
+        versionCode computeVersionCode()
+        versionName computeVersionName()
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
+    signingConfigs {
+      ci {
+          storeFile file(System.env.CI_KEYSTORE_PATH ?: "../key/debug.keystore")
+          storePassword System.env.CI_KEYSTORE_PASS ?: "BOaMocmwQ1qiq9gMIhcHBEYgE"
```

Comment:
We use separate certificates for the productions builds. They are unique for each projects. The only thing an attacker can do is build an application that looks like an develop application version. Otherwise, we will force developers to store this file locally and set environment variables to run a local build.

Reply:
We will!